libraries/core-react/src/components/Typography/Typography.test.tsx

```diff
@@ -148,4 +148,16 @@ describe('Typography', () => {
     expect(typography).toHaveStyleRule('margin-top', '16px')
     expect(typography).toHaveStyleRule('margin-bottom', '32px')
   })
+  it('can focus links', () => {
+    render(
+      <Typography link href="#">
+        Link
+      </Typography>,
+    )
+
+    const typography = screen.getByText('Link')
+
+    typography.focus()
+    expect(typography).toHaveFocus()
```

Comment:
This test does not test much... It would still be green even if the outline was wrong or not present. Maybe add some more assertions on style?